changelog.d/20250319_182641_meilisearch_host.md

```diff
@@ -0,0 +1,14 @@
+<!--
+Create a changelog entry for every new user-facing change. Please respect the following instructions:
+- Indicate breaking changes by prepending an explosion 💥 character.
+- Prefix your changes with either [Bugfix], [Improvement], [Feature], [Security], [Deprecation].
+- You may optionally append "(by @<author>)" at the end of the line, where "<author>" is either one (just one)
+  of your GitHub username, real name or affiliated organization. These affiliations will be displayed in
+  the release notes for every release.
+-->
+
+- [Improvement] This change introduces MEILISEARCH_HOST so that it can be
+re-used in the project, this also enables Tutor to configure meilisearch host
+and expose it to be used by other plugins. For example MEILISEARCH_HOST can be
+optinonally used to configure ingress controller in openedx-k8s-harmony like
+LMS_HOST etc.
```

Comment:
> so that it can be
re-used in the project,

I don't think this wording is helpful. It is implicit that the configs are re-usable. The need of the config, which was explained later, should be good enough.

nit: optinonally --> optionally.

Reply:
thanks and you are right.